db/migrate/20230713095930_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true, on_delete: :cascade
```

Comment:
- [ ] The foreign key to the user's table should be named `author_id` as you did for the posts table.